test/IDE/complete_at_top_level.swift

```diff
@@ -471,7 +471,7 @@ var stringInterp = "\(#^STRING_INTERP_3^#)"
 _ = "" + "\(#^STRING_INTERP_4^#)" + ""
 // STRING_INTERP: Begin completions
 // STRING_INTERP-DAG: Decl[InstanceMethod]/CurrNominal/IsSystem: ['(']{#(value): T#}[')'][#Void#];
-// STRING_INTERP-DAG: Decl[Struct]/CurrModule: FooStruct[#FooStruct#];
+// STRING_INTERP-DAG: Decl[Struct]/CurrModule/TypeRelation[Convertible]: FooStruct[#FooStruct#]; name=FooStruct
```

Comment:
@rintaro I tracked down the real difference and I think this is it. Does that make sense, given we've overloaded interpolation to take a metatype?

Reply:
Yeah, it makes sense 👍